packages/strapi-admin/index.js

```diff
@@ -279,9 +280,14 @@ async function watchFiles(dir) {
     const packageName = isExtension
       ? `strapi-plugin-${pluginName}`
       : 'strapi-admin';
+
+    const spliter = os.platform() === 'win32' ? '\\' : '/';
```

Comment:
Instead of using separators manually (you can use path.sep btw) Can you please use path.relative to compite the target path instead of with split and replace ? Thanks !

Reply:
path.sep is a good idea. thanks!

Reply:
The second part,
I just want to fix this bug and not going to improve this part at this moment.
And I don't want to change the original logic.
Also I don't think `path.relative` able to improve this part.
Thanks.